packages/website/builder/css/nav.css

```diff
@@ -165,7 +163,7 @@
 
 .nav_toggle,
 .nav_toggle-label::before {
-  font-family: "Courier New", monospace;
+  font-family: monospace, monospace;
```

Comment:
I think one `monospace` should be enough.

Reply:
No, there's a bug and this is the hack fix. One monospace does the wrong thing in various browsers which misbehave.

Reply:
I see.